source/supported-operations.txt

```diff
@@ -345,8 +347,8 @@ Date and Time Functions
    * - ``MAKEDATE()``
      - Create a date from the year and day of year
    * - ``MICROSECOND()``
-     - Return the microseconds from the time or datetime expression 
-       ``<expr>`` as a number in the range from 0 to 999999.
+     - Return the microseconds from the input time or datetime expression 
+       as a number between from 0 to 999999 inclusive.
```

Comment:
Typo here - we have both "between" and "from".

Either is fine, but not both! :D